charts/k6-operator/values.yaml

```
@@ -111,6 +111,32 @@ prometheus:
   # prometheus.enabled -- enables the prometheus metrics scraping (default: false)
   enabled: false
 
+# @schema
+# required: false
+# type: object
+# @schema
+service:
+  # @schema
+  # required: false
+  # type: boolean
+  # @schema
+  # service.enabled -- enables the k6 service (default: false)
+  enabled: false
```

```suggestion
  enabled: true
```
Current default state for Service is actually true. Let's not change the default without a reason.